docs/concepts/index.md

```diff
@@ -10,27 +10,30 @@ tags:
 # Explore Prefect concepts
 
 ## Develop
-* [Flows](flows.md)
-* [Tasks](tasks.md)
-* [Results](results.md)
-* [Artifacts](artifacts.md)
-* [States](states.md)
-* [Blocks](blocks.md)
-* [Variables](variables.md)
-* [Task Runners](task-runners.md)
-* [Runtime Context](runtime-context.md)
-* [Profiles & Configuration](settings.md)
 
-##  Deploy
-* [Deployments](deployments.md)
-* [Projects](projects.md)
-* [Work Pools, Workers & Agents](work-pools.md)
-* [Storage](storage.md)
-* [Filesystems](filesystems.md)
-* [Infrastructure](infrastructure.md)
-* [Schedules](schedules.md)
-* [Logging](logs.md)
+| Keyword                                            | Description                                                                                                                 |
+| -------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------- |
+| __[Flows](/concepts/flows)__                       | A Prefect workflow, modeled as a Python function.                                                                           |
+| __[Tasks](/concepts/tasks)__                       | Discrete units of work in a Prefect workflow.                                                                               |
+| __[Results](/concepts/results)__                   | The data returned by a flow or a task.                                                                                      |
+| __[Artifacts](/concepts/artifacts)__               | Persisted outputs for human viewing such as tables or links.                                                                |
```

Comment:
```suggestion
| __[Artifacts](/concepts/artifacts)__               | Formatted outputs rendered in the Prefect UI, such as markdown, tables, or links.                                                                |
```

Reply:
@gabcoyne It would be cool to get this PR into the release today. Thoughts on the above? 👽